### ISSUE TITLE
NuScenes prediction Benchmark 

### ISSUE THREAD

**Issue by mo-zahran** [nutonomy/nuscenes-devkit#375]
Hi Team
I have two questions related prediction task:

**1. How to reproduce the numbers in CoverNet paper and MTP using the same [Evaluation metrics](https://www.nuscenes.org/prediction/#evaluation-metrics) in the challenge?** 
So the released code (as far as I now) supports only inference, but I cannot find the evaluation code. For example, If I want to calculate minADE_k or minFDE_k for CoverNet to reproduce the same results reported on the paper, how should I do this? Is there an open-source implementation for **Evaluation metrics** that will be used in the competition and how can we use it on CoverNet or MTP 

**2. What is the possibility to open another submission in the challenge for validation submissions?**
So some competitions have two leaderboards, one for validation dataset and one for testing. The validation leaderboard is open to unlimited submission times and testing is limited to a specific number as the current situation. This will help the community to find better results given that the testing and validation data are I.I.D.

**Comment by freddyaboulton** [comment]
Hi @mo-zahran , thanks for the question! 

1. The evaluation metrics are defined in this script (https://github.com/nutonomy/nuscenes-devkit/blob/master/python-sdk/nuscenes/eval/prediction/metrics.py) You can use them to evaluate your model during training or inference. These are the same metrics reported in the paper (with the exception that we turned the hit rate into the miss rate)

2. The leaderboard right now is for the validation set. I believe it is configured to only accept 3 submissions per team but we released the annotations for the validation set so you can run evaluation locally as many times as you'd like. Here is a script showing how to do that (https://github.com/nutonomy/nuscenes-devkit/blob/master/python-sdk/nuscenes/eval/prediction/compute_metrics.py)
